tests/adapter/dbt/tests/adapter/utils/base_utils.py

```diff
@@ -1,10 +1,17 @@
 import pytest
 from dbt.tests.util import run_dbt
 
+macros__equals_sql = """
+{% macro equals(actual, expected) %}
+{# -- actual is not distinct from expected #}
+(({{ actual }} = {{ expected }}) or ({{ actual }} is null and {{ expected }} is null))
```

Comment:
I really despise how the "absence of value is not equal to itself" definition is formally correct but does not reflect how anyone uses a Null in practice. Maybe we need a ANSI sequel patch while we're at it 😁